server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -484,11 +452,10 @@ static Map<String,DataFileValue> cleanupRootTabletFiles(VolumeManager fs, String
   public void upgradeFileDeletes(ServerContext context, Ample.DataLevel level) {
 
     String tableName = level.metaTable();
-    AccumuloClient c = context;
```

Comment:
Why remove this and then cast below?

Reply:
fixed with 1a5c6cc055

Reply:
This doesn't appear to be fixed.

Reply:
This removes the declaration of `AccumuloClint c = ...` that is used only in one place.  This eliminates that variable and make the cast explicit only where is is needed.